tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -685,7 +685,8 @@ EOF
 		fi
 		export RUSTFLAGS
 		# Foreign CC is incompatible with libgit2 -- CC is still handled by `-C linker=...` flag
-		CC= cargo build --release --target "${target}" --no-default-features --features "${AA_KBC}"
+		# AA_KBC=cc_kbc: tdx-attest-sys build fails unless a value is provided for SGX_SDK
+		CC= SGX_SDK= cargo build --release --target "${target}" --no-default-features --features "${AA_KBC}"
```

Comment:
Don't we need to build the attestation-agent from app/ directory (as make is doing) and also with some more features?
E.g.:
```
pushd app
CC= SGX_SDK= cargo build --release --target "${target}" --no-default-features --features "${AA_KBC},grpc,rust-crypto"
popd
```